tests/test_cli.py

```diff
@@ -373,6 +374,7 @@ def test_component_build(mock_build, mock_push):
         pull=True,
         target="base",
         decode=True,
+        labels={"label_0_key": "label_0_value", "label_1_key": "label_1_value"},
```

Comment:
```suggestion
        label={"label_0_key": "label_0_value", "label_1_key": "label_1_value"},
```

Reply:
This one should be indeed `label**s**`. In the docker build api it is called `labels` as well.